### PR TITLE
add simple labels to the images

### PR DESCRIPTION
**Pull request by Emilgardis**

### CODE REVIEW

**Comment by Emilgardis** [comment]
bors try --target x86_64-*

**Comment by Emilgardis** [comment]
bors cancel

locals!

**Comment by Emilgardis** [comment]
is canceling broken?

@bors r-

**Comment by Emilgardis** [comment]
bors try --target x86_64-*

**Comment by Emilgardis** [comment]
git is not cooperating with me :)

**Comment by Emilgardis** [comment]
bors try --target x86_64-*

File: build-docker-image.sh [hunk 1]
```diff
@@ -79,6 +79,10 @@ run() {
     build_args+=(--tag "${tag}")
   done
 
+  if [ -n "${LABELS}" ]; then
+    build_args+=(--label "${LABELS}")
```
[the remark this answers is not in the document]
Comment (Emilgardis) [reply build-docker-image.sh:83]: `--help` says `--label stringArray`, so comma separated labels (`--label foo=baz,bar=foo`) works

[the remark this answers is not in the document]
Comment (Alexhuszagh) [reply build-docker-image.sh:83]: This doesn't work, just tested it. I created a simple image, built it as follows, and inspected the labels:

```bash
$ docker build --label foo=baz,bar=foo --label baz=bar --tag "labels:latest" .
$ docker inspect labels:latest -f {{.Config.Labels}}
map[baz:bar foo:baz,bar=foo]
```

**Comment by Emilgardis** [comment]
bors try --target x86_64-unknown-linux-gnu

**Review by Alexhuszagh — CHANGES_REQUESTED** [review]
The metadata separation of labels using commas doesn't work, actually, so this will need to be refactored.

I tried the following with the sample `Dockerfile`:

```Dockerfile
FROM ubuntu:22.04
RUN apt-get update && apt-get install -y wget
```

Then, built and inspected as follows:

```bash
$ docker build --label foo=baz,bar=foo --label baz=bar --tag "labels:latest" .
$ docker inspect labels:latest -f {{.Config.Labels}}
map[baz:bar foo:baz,bar=foo]
```

Or, we have only 2 labels, not 3. We have `baz` with a value of `bar`, and `foo` with a value of `baz,bar=foo`.

**Comment by Emilgardis** [comment]
bors try --target x86_64-unknown-linux-gnu

**Review by Alexhuszagh — APPROVED** [review]

**Comment by Alexhuszagh** [comment]
Ok once all the commits are squashed, feel free to merge this with r=me.

**Comment by Emilgardis** [comment]
bors try --target x86_64-*

one last time, want to ensure it looks good

**Comment by Emilgardis** [comment]
bors try --target x86_64-*

**Comment by Alexhuszagh** [comment]
bors try --target x886_64-pc-windows-msvc

**Comment by Alexhuszagh** [comment]
bors try --target x86_64-pc-windows-msvc

**Review by Alexhuszagh — APPROVED** [review]

**Comment by Emilgardis** [comment]
ah crap, my fault

**Pull request merged**